easybuild/easyconfigs/l/libvips/libvips-8.9.2-foss-2019a.eb

```
@@ -0,0 +1,40 @@
+easyblock = 'ConfigureMake'
+
+name = 'libvips'
+version = '8.9.2'
+
+homepage = 'https://libvips.github.io/libvips/'
+description = """libvips is a demand-driven, horizontally threaded image processing library."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+github_account = 'libvips'
+source_urls = ['https://github.com/libvips/libvips/releases/download/v%(version)s']
+sources = ['vips-%(version)s.tar.gz']
+checksums = ['ae8491b1156cd2eb9cbbaa2fd6caa1dc9ed3ded0b70443d28cd7fea798ab2a27']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('binutils', '2.31.1'),
+]
+
+dependencies = [
+    ('libpng', '1.6.36'),
+    ('libjpeg-turbo', '2.0.2'),
+    ('LibTIFF', '4.0.10'),
+    ('giflib', '5.1.4'),
+    ('FFTW', '3.3.8'),
+    ('GLib', '2.60.1'),
+    ('expat', '2.2.6'),
+    ('librsvg', '2.48.4'),
+    ('ImageMagick', '7.0.8-46'),
+    ('OpenSlide', '3.4.1', '-largefiles'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/vips', 'bin/batch_image_convert', 'bin/vipsthumbnail', 
```

Tests fails due to this white space
```suggestion
    'files': ['bin/vips', 'bin/batch_image_convert', 'bin/vipsthumbnail',
```

---

Since this is trivial, I'm committing the code style fix myself :slightly_smiling_face: